pkg/scheduler/metrics/profile_metrics.go

```diff
@@ -44,5 +44,6 @@ func PodScheduleError(profile string, duration float64) {
 
 func observeScheduleAttemptAndLatency(result, profile string, duration float64) {
 	e2eSchedulingLatency.WithLabelValues(result, profile).Observe(duration)
+	schedulingLatency.WithLabelValues(result, profile).Observe(duration)
```

Comment:
I'm not super comfortable with setting a metric to stable immediately in the same PR that it is introduced.. can we bake this a little?

Reply:
Fine with me. FYI @vantuvt

Reply:
@logicalhan, in case it wasn't clear, this is just a "rename" by duplication.

Reply:
@logicalhan Also wanted to point out that this kind of renaming+making stable has been done before https://github.com/kubernetes/kubernetes/pull/99785

Reply:
I also don't think we should go through another round of Alpha stability when adjusting a metric before it graduates to stable. As a matter of fact, Kubernetes resources can also be modified during the graduation process to GA.

Reply:
ok ok...